js/app.js

```diff
@@ -1,11 +1,32 @@
-angular.module('PathOfDamage', [])
-.controller('Damage', function ($scope, $document, $window, DataService) {
+angular.module('PathOfDamage', ['ngSanitize', 'ui.select'])
+.controller('Damage', function ($scope, $document, $window, DataService, Items) {
+  const ELEMENTS = ['physical', 'fire', 'cold', 'lightning', 'chaos'];
```

Comment:
I don't want to duplicate this const, but I think the correct way is to pass it into the DataService during its construction probably, and that tripped me up when I was messing with it.